src/components/ImageTextBlock/ImageTextBlockStyles.scss

```diff
@@ -0,0 +1,54 @@
+@import "../../styles/variables";
+@import "../../styles/breakpoints.scss";
+
+.image-text-block {
+  position: relative;
+  display: flex;
+
+  &__text {
+    flex: 1;
+    padding-right: $spacing-large;
+
+    @include breakpoint('mobile') {
+      padding: 0;
+    }
+
+    &-label {
+      text-decoration: underline;
+    }
+  }
+
+  &__image {
+    flex: 1;
+  }
+
+  &:nth-of-type(even) {
+    flex-direction: row-reverse;
+
+    .image-text-block__text {
+      text-align: right;
+      padding-right: 0;
+      padding-left: $spacing-large;
+
+      @include breakpoint('mobile') {
+        padding: 0;
+        text-align: left;
+      }
+    }
+  }
+
+  & + & {
```

Comment:
i haven't seen this before but curious what is this & + & syntax doing?

Reply:
Ahhh yeaahh, the `+` selector is a personal fave and used for selecting adjacent siblings! So `div + p` would select the first `p` that follows a `div`
```
<>
    <div>Not selected</div>
    <p>Selected!</p>
</>
```

The `&` selector is scss-specific and grabs the parent selector in the current nested scope, super convenient so you don't need to keep typing out the parent class over and over again.
```
.header {
   color: red;

   &__logo { // this would translate to '.header__logo'
      ...
   }
}
``` 


So in the highlighted line, the `& + &` is just `.image-text-block + .image-text-block` and gives any `.image-text-block` (that isn't the first) a margin-top. Avoids having to create that extra definition of `.image-text-block:first-of-type { margin-top: none; }`